packages/core/src/bundle.ts

```diff
@@ -224,13 +214,23 @@ export function mapTypeToComponent(typeName: string, version: OasMajorVersion) {
         default:
           return null;
       }
+    case SpecMajorVersion.Async2:
+      switch (typeName) {
+        case 'Schema':
+          return 'schemas';
+        // TODO + verify
```

Comment:
What exactly needs to be verified here?

Reply:
I wanted to double check if async api has schemas, I believe it does so it can be removed.

Reply:
Thanks! Removed